build-tests/api-extractor-test-01/package.json

```diff
@@ -17,5 +17,9 @@
     "@types/node": "8.5.8",
     "fs-extra": "~7.0.1",
     "typescript": "~3.1.6"
+  },
+  "devDependencies": {
+    "@types/long": "^4.0.0",
```

Comment:
`@types` packages must never use `^` or `~` in their version specifier because they do not follow SemVer.  The version number is meant to follow the version of the underlying JavaScript package.  Thus a patch-level increment can break the API contract (e.g. if someone decides to rename an interface).